views/LocationListView/LocationListView.tsx

```diff
@@ -48,12 +57,15 @@ export const LocationListView = () => {
             />
           ))
         ) : (
-          <Typography
-            variant="header-medium"
-            text="Nie masz zadnych lokalizacji"
-          />
+          <Typography variant="header-medium" text="Brak zapisanych lokacji" />
         )}
         <Button text="Stwórz nową lokalizację" onPress={onNavigateToAdmin} />
+        {botsAvailable && (
+          <Button
+            text="Załaduj dostępne lokacje"
+            onPress={retrieveAvailableLocations}
+          />
+        )}
```

Comment:
I don't see any reason to add this button

Reply:
I will remove it